src/MIDebugEngine/AD7.Impl/AD7Engine.cs

```diff
@@ -1083,6 +1082,19 @@ public int WatchForThreadStep(IDebugProgram2 pOriginatingProgram, uint dwTid, in
 
         #endregion
 
+        #region IDebugEngineProgram2 Members
+        int IDebugReversibleEngineProgram160.CanReverse()
+        {
+            return DebuggedProcess.TargetFeatures.Contains("reverse") ? 1 : 0;
```

Comment:
The `int` here is an `HRESULT`. 

You will want to return `Constants.S_OK` if `DebuggedProcess.TargetFeatures.Contains("reverse")` is true, and `Constants.S_FALSE` otherwise.